<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RS232</class>
 <widget class="QWidget" name="RS232">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Show text as:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="text_type_combobox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>ASCII</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hex</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="text">
     <property name="minimumSize">
      <size>
       <width>675</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Mono</family>
      </font>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Input:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="input_combobox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>100</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtBottom</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="line_ending_combobox">
       <property name="toolTip">
        <string>Line Ending</string>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>\n</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>\r</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>\r\n</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>\n\r</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>\0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hex:</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_ending_lineedit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Baudrate:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save Configuration</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Hardware Flow Control:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Word Length:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="stopbits_spinbox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Parity:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="wordlength_spinbox">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="parity_combobox">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Odd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Even</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Forced Parity 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Forced Parity 0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="baudrate_combobox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>300 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>600 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1200 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14400 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28800 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>230400 baud</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Stop Bits:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="hardware_flowcontrol_combobox">
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Software Flow Control:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="software_flowcontrol_combobox">
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>text_type_combobox</tabstop>
  <tabstop>text</tabstop>
  <tabstop>input_combobox</tabstop>
  <tabstop>baudrate_combobox</tabstop>
  <tabstop>parity_combobox</tabstop>
  <tabstop>stopbits_spinbox</tabstop>
  <tabstop>wordlength_spinbox</tabstop>
  <tabstop>hardware_flowcontrol_combobox</tabstop>
  <tabstop>software_flowcontrol_combobox</tabstop>
  <tabstop>save_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
